app/selectors/autocomplete.js

```diff
@@ -258,14 +258,18 @@ export const filterDirectAndGroupMessages = createSelector(
         if (matchTerm === null) {
             return null;
         }
+        let mt = matchTerm;
+        if (matchTerm.startsWith('@')) {
+            mt = matchTerm.replace(/^@/, '');
```

Comment:
you know that `matchTerm` starts with and `@` sign, so using a substr is faster than replacing with a regex so you could do

```suggestion
            mt = matchTerm.substr(1);
```